docker/k8s/fabric/README.md

```diff
@@ -0,0 +1,61 @@
+# Kubernetes playground for Avalon Fabric proxy model
+
+This serves as a playground to setup k8s cluster for Avalon Fabric proxy model.
+
+## Prerequisites
+- To setup the k8s cluster using minikube, refer to Prerequisites section in README.md under docker/k8s folder
```

Comment:
Add a link to README.md in the parent directory:

```
To setup the k8s cluster using minikube, refer to Prerequisites section in
 [README.md](../README.md) under docker/k8s folder
```

Reply:
Done